app/views/sections/destination/DestinationBusinessNameView.scala.html

```diff
@@ -0,0 +1,55 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.requests.DataRequest
+@import viewmodels.InputWidth._
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukInput: GovukInput,
+    govukButton: GovukButton,
+    h1: components.h1,
+    continueOrExit: components.continueOrExit
+)
+
+@(form: Form[_], onSubmitCall: Call)(implicit request: DataRequest[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("destinationBusinessName.title"))) {
+
+    @formHelper(action = onSubmitCall) {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form))
+        }
+
```

Comment:
```suggestion

@h2(messages("section.destination"), "govuk-caption-xl")
```
(it'd be better if this is `destination.subHeading` to align with the others but that's just a nitpick, not a priority)